apps/Gruntfile.js

```diff
@@ -24,6 +24,7 @@ module.exports = function (grunt) {
       `require('${path.resolve(process.env.mocha_entry)}');`;
     const file = `// Auto-generated
 import 'babel-polyfill';
+import 'whatwg-fetch';
 import { throwOnConsoleErrorsEverywhere, throwOnConsoleWarningsEverywhere } from './util/testUtils';
```

Comment:
`whatwg-fetch` is also referenced here: https://github.com/code-dot-org/code-dot-org/blob/621db84cc5e1a7b2506c449d4a89d15d68697c75/apps/Gruntfile.js#L542. We need it in both places?

Reply:
Yes - line 542 in the gruntfile makes sure fetch is polyfilled in our built bundles.  Line 27 adds it to the entry tests template, so it's also available when running a subset of unit tests.